pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
@bennofs pytest is only accessible via a package set like pythonPackages and not as a single package. I would say the current approach chosen here is ok and does not deviate from other python applications.